.mvn/maven.config

```diff
@@ -0,0 +1,2 @@
+-Pconsume-incrementals
```

Comment:
not related should be part of an other pr

Reply:
Related I wanted the PR to be incremental so I could use it in jcasc